Langs/0410.lang

```diff
@@ -14,71 +14,90 @@ This new Enhanced Accessibility functionality is designed for the visually impai
 Results Based on Currently Known Requirements!=Risultati basati sui requisiti di sistema attualmente conosciuti!
+Language=Lingua
+Theme=Tema
 Translation by=Traduzione di
-Architecture=Architettura (CPU + SO)
-Boot Method=Metodo Boot
+Theme by=Autore tema
+Architecture=Architettura (CPU + S.O.)
+Boot Method=Metodo boot
 CPU Compatibility=Compatibilità CPU
 CPU Core Count=Core CPU
 CPU Frequency=Frequenza CPU
 DirectX + WDDM2=DirectX + WDDM2
-Disk Partition Type=Tipo di partizione del disco
-RAM Installed=RAM Installata
+Disk Partition Type=Tipo partizione disco
+RAM Installed=RAM installata
 Secure Boot=Secure Boot
 Storage Available=Archiviazione disponibile
 TPM Version=Versione TPM
-Checking...=Verifica in corso...
+Checking...=Verifica...
 64 Bit CPU=CPU a 64 Bit
-64 Bit OS=SO a 64 Bit
+64 Bit OS=Sostema operativo a 64 Bit
 32 Bit CPU=CPU a 32 Bit
-32 Bit OS=SO a 32 Bit
+32 Bit OS=Sistema operativo a 32 Bit
 Unable to Determine=Impossibile da determinare
-Unable to Check List=Impossibile verificare la lista
-Error Accessing List=Errore durante l'accesso alla lista
+Unable to Check List=Impossibile verificare l'elenco
+Error Accessing List=Errore durante l'accesso all'elenco
+Check Timed Out=Tiomepout verifica
+Check Skipped=Verifica saltata
 Not Currently Listed as Compatible=Attualmente non elencato come compatibile
 Listed as Compatible=Elencato come compatibile
 Cores=Core
 Threads=Thread
-GPT Detected=GPT individuato
-GPT Not Detected=GPT non individuato
-Enabled=Abilitato
-Supported=Supportato
-Not Supported=Non supportato
-Disabled / Not Detected=Disabilitato / Non rilevato
+GPT Detected=Rilevato GPT
+GPT Not Detected=Non rilevato GPT
+Enabled=abilitato
+Supported=supportato
+Not Supported=non supportato
+Disabled / Not Detected=Disabilitato/non rilevato
 Drive(s) Meet Requirements=unità soddisfano i requisiti
-0 Drives Meet Requirements=0 unità soddisfano i requisiti
-TPM Missing / Disabled=TPM Mancante / Disabilitato
-Detected=Individuato
+0 Drives Meet Requirements=Nessuna unità soddisfa i requisiti
+TPM Missing / Disabled=TPM non presente o disabilitato
+Detected=Rilevato
+May Report DirectX 12 Incorrectly=Potrebbe segnalare DirectX 12 in modo errato
 DirectX 12 and WDDM 3=DirectX 12 e WDDM 3
 DirectX 12 and WDDM 2=DirectX 12 e WDDM 2
 No DirectX 12, but WDDM2=DirectX 12 mancante ma WDDM 2 presente
 DirectX 12, but no WDDM2=DirectX 12 presente ma WDDM 2 mancante
 No DirectX 12 or WDDM2=DirectX 12 e WDDM 2 mancanti
+DxDiag Errored=Errore DxDiag
 Test Build?=Build di test?
-You're running a newer build than publicly Available!=Stai eseguendo una versione più recente di quella disponibile pubblicamente!
+You're running a newer build than publicly Available!=Questa versione è più recente di quella disponibile pubblicamente!
 Up to Date=Aggiornato
-You're running the latest build!=Stai usando la versione più recente!
+You're running the latest build!=Questa versione è aggiornata!
```

Comment:
What's difference between this and the old one? Because this translated to "This version is updated" can have different meaning. Unless that's how say it in Italian.

Reply:
In standard software translation is unsual to use reflessive verb kike "Stai usando".
As is unusal use of "più recente" and also not related to english string . 
For most user it's more clear to understand "versione aggiornata".
I changed it as "La versione del programma è aggiornata."